molecule/azkabanwebsolo/requirements.yml

```diff
@@ -0,0 +1,9 @@
+- src: https://github.com/idealista/mysql-role.git
+  scm: git
+  version: 2.2.2
+  name: mysql
+
+- src: https://github.com/idealista/java-role.git
```

Comment:
I would prefer to declare the src with underscore to avoid redirections: `java_role`

Reply:
Done